openthread.sln

```diff
@@ -0,0 +1,79 @@
+﻿
```

Comment:
Would you mind if I left the solution file at the root? It greatly simplifies the output directory copying and linking between projects.